src/overlays/actors/ovl_En_Tk/z_en_tk.c

```diff
@@ -1189,7 +1189,7 @@ void func_80AEED38(EnTk* this, PlayState* play) {
         this->actor.shape.rot.y = this->actor.world.rot.y;
     }
 
-    if (Message_GetState(&play->msgCtx) == 0 && !Play_InCsMode(play) && (this->unk_2C6-- <= 0)) {
+    if (Message_GetState(&play->msgCtx) == TEXT_STATE_NONE && !Play_InCsMode(play) && (this->unk_2C6-- <= 0)) {
```

Comment:
```suggestion
    if ((Message_GetState(&play->msgCtx) == TEXT_STATE_NONE) && !Play_InCsMode(play) && (this->unk_2C6-- <= 0)) {
```
I don't suppose the last bit can be split up to not have the -- in the comparison?